libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/entitlements/FilesEntitlement.java

```diff
@@ -160,7 +163,7 @@ public FileData withPlatform(Platform platform) {
 
         @Override
         public String description() {
-            return Strings.format("[%s] <%s>/%s%s", mode, baseDir, relativePath, exclusive ? " (exclusive)" : "");
+            return Strings.format("[%s] <%s>%s%s%s", mode, baseDir, SEPARATOR, relativePath, exclusive ? " (exclusive)" : "");
```

Comment:
What's the significance of this?

Reply:
To have separators consistent in the output; on Windows that was looking weird, like `<DATA>/c\d`
This is more cosmetic, but since I fixed the assertion I wanted it to be consistent too.